src/jitlayers.cpp

```diff
@@ -203,7 +209,6 @@ void addOptimizationPasses(legacy::PassManagerBase *PM, int opt_level,
     // Run instcombine after redundancy elimination to exploit opportunities
     // opened up by them.
     PM->add(createSinkingPass()); ////////////// ****
-    PM->add(createInstructionSimplifierPass());///////// ****
```

Comment:
Why was this removed?

Reply:
Fixed, now it has been removed only for LLVM 7 (it's a subset of InstructionCombiningPass).